pkgs/tools/package-management/libdnf/default.nix

```diff
@@ -0,0 +1,62 @@
+{ gcc9Stdenv, stdenv, fetchFromGitHub, cmake, gettext, pkg-config, gpgme, libsolv, openssl, check, 
+pcre, json_c, libmodulemd, libsmartcols, sqlite, librepo, libyaml, rpm }:
+
+gcc9Stdenv.mkDerivation rec {
+  pname = "libdnf";
+  version = "0.55.2";
+
+  src = fetchFromGitHub {
+    owner = "rpm-software-management";
+    repo = pname;
+    rev = version;
+    sha256 = "0hiydwfa90nsrqk5ffa6ks1g73wnsgjgq7z7gwq9jj76a7gpfbfq";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    gettext
+    pkg-config
+    check
+    gpgme
+    openssl
+    json_c
+    libsmartcols
+    libyaml
+    libmodulemd
```

Comment:
These should be regular `buildInputs` – `nativeBuildInputs` are programs that need to run on the _build_ system, `buildInputs` are for the _host_ system (when cross-compiling, http://gcc.gnu.org/onlinedocs/gccint/Configure-Terms.html).